README.md

```diff
@@ -28,6 +28,21 @@ Proudly brought to you by [@revolunet](http://twitter.com/revolunet), [@deltaeps
 
 Or alternatively, grab the dist/angular-google-analytics.min.js and include it in your project
 
+
+In your html/template add
+```html
+...
+<script src="bower_components/angular-google-analytics/dist/angular-google-analytics.js"></script>
+...
+```
+
+In your application, declare dependency injection like so..
```

Comment:
Please change this to be: "In your application, declare the `angular-google-analytics` module dependency."

Please remove the "..." from the code segments as well since that doesn't fit with the style used throughout the document.